docs/Air.md

```diff
@@ -437,6 +437,23 @@ Gets pnr information and tickets list from [`importPNR`](#importPNR) and then do
 
 **See: <a href="../examples/Air/cancelBooking.js">cancelBooking example</a>**
 
+## .providerReservationDivide(params)
+<a name="providerReservationDivide"></a>
+
+Sends a pnr divide request and returns parent and child reservation information with divided passenger list.
+
+**Returns**: `Promise` 
+
+**See**: [Dividing Universal Records](https://support.travelport.com/webhelp/uapi/uapi.htm#Booking/UniversalRecord/Dividing_Universal_Records.htm%3FTocPath%3DUniversal%2520Records%2520and%2520Bookings%7C_____5)
+and <a href="../examples/Air/divide.js">providerReservationDivide example</a>**
+
+| Param | Type | Description |
+| --- | --- | --- |
+| uapi_ur_locator | `Object` | Universal Record Locator |
```

Comment:
It's not an object.
Also, could we allow to skip passing this locator as w we might get it internally?

Reply:
And could you please add a description of the response?